sushy/interop_profiles/ServiceRoot.v1_0_0.json

```diff
@@ -0,0 +1,73 @@
+{
+	"SchemaDefinition": "RedfishInteroperabilityProfile.v1_0_0",
+	"ProfileName": "RedfishProfile",
+	"ProfileVersion": "1.0.0",
+	"Purpose": "Redfish Interoperability profile",
+	"OwningEntity": "Dell",
+	"ContactInfo": "abc@dell.com",
+	"Resources": {
+		"ServiceRoot": {
+			"PropertyRequirements": {
+				"Chassis": {
+				        "Comparison":"LinkToResource",
+                                        "Values":["ChassisCollection"]
```

Comment:
I would remove "Comparison" and "Values" from this since the schema rules for ServiceRoot already enforce that "Chassis" is a link to a ChassisCollection. The only time you'd want to show this type of rule is for a fungible link like "PoweredBy", which can point to multiple types of resources.

Reply:
The same applies to all other cases where Comparison and Values are being used in this file. They aren't needed since all of the links in this resource are hardened to a single type.

Reply:
Noted and thanks. Made required changes in the profile.